service/src/main/java/com/commercetools/pspadapter/payone/ScheduledJob.java

```diff
@@ -42,17 +41,24 @@ public void execute(final JobExecutionContext context) throws JobExecutionExcept
 
             final Consumer<Payment> paymentConsumer = payment -> {
                 try {
-                    final PaymentWithCartLike paymentWithCartLike = queryExecutor.getPaymentWithCartLike(payment.getId(), CompletableFuture.completedFuture(payment));
+                    final PaymentWithCartLike paymentWithCartLike = queryExecutor
+                        .getPaymentWithCartLike(payment.getId(), CompletableFuture.completedFuture(payment));
                     paymentDispatcher.dispatchPayment(paymentWithCartLike);
                 } catch (final NoCartLikeFoundException ex) {
-                    LOG.debug(LOG_PREFIX + " Could not dispatch payment with ID \"{}\": {}",
-                            tenantFactory.getTenantName(),  payment.getId(), ex.getMessage());
+                    LOG.error(
+                        createTenantKeyValue(tenantFactory.getTenantName()),
+                        format("Could not dispatch payment with id '%s'", payment.getId()),
+                        ex);
```

Comment:
I am not sure it is useful to see a stack trace in case cart is not found. Why following
`Could not dispatch payment with id 123 as its cart could not be found.` is not enough?

Reply:
the thing is the google error reporting only works (only renders a log as an error in the error reporting when it has a stack trace otherwise, it's not picked up in the error reporting errors).

https://cloud.google.com/error-reporting/docs/formatting-error-messages